src/net/smooth/segment_loader.ts

```diff
@@ -127,12 +123,14 @@ const generateSegmentLoader = (
       value: { responseData },
     });
   }
+  else if (segment.mediaURL == null) {
+    return observableOf({
+      type: "data" as "data",
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this unnecessary cast. ([typescript:S4325](https://sonarcloud.io/organizations/rx-player/rules#rule_key=typescript%3AS4325))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=rx-player&pullRequest=237&issues=AWPRFuZd4427HkKVonsb&open=AWPRFuZd4427HkKVonsb)

[ISSUE_KEY]: AWPRFuZd4427HkKVonsb